src/VirtualCollection.vue

```diff
@@ -55,62 +55,77 @@ export default {
     },
     data() {
         return {
-            displayItems: [],
             totalHeight: 0,
-            totalWidth: 0
+            totalWidth: 0,
+            displayItems: [],
+            groupManagers: []
```

Comment:
The `groupManagers` doesn't need to be bind. Can you move it to the `created()` method ? This can fix the flaky test.